tests/perception/high_level_semantics_situation_to_developmental_primitive_perception_test.py

```diff
@@ -246,3 +248,18 @@ def test_learner_perceivable():
     ).frames[0]
 
     assert _some_object_has_binary_property(learner_perception, IS_LEARNER)
+
+
+def test_ground():
```

Comment:
Also add a test to ensure that if the ground is explicitly specified in a situation, it is not added a second time